app/src/main-process/menu/build-default-menu.ts

```diff
@@ -331,7 +331,9 @@ export function buildDefaultMenu({
         label: __DARWIN__
           ? 'Create Issue on GitHub'
           : 'Create &issue on GitHub',
-        accelerator: 'CmdOrCtrl+Shift+I',
+        accelerator: (() => {
+          return __DARWIN__ ? 'Cmd+Shift+I' : 'Ctrl+I'
```

Comment:
I appreciate not changing more than we have to, but since this is a relatively new accelerator, lets keep them consistent across platforms! (And now we can drop the self-executing anonymous function here, too.)

```suggestion
          return 'CmdOrCtrl+I'
```